src/sklearndf/wrapper/_wrapper.py

```diff
@@ -616,7 +617,15 @@ def __setattr__(self, name: str, value: Any) -> None:
         if name.startswith("_"):
             super().__setattr__(name, value)
```

Comment:
Shouldn't we raise an exception here? I see that the comment indicates that only public attributes can be set here.
Right now it's the same as in line `628`. Also `__getattr__` raises an exception for accessing private attributes (line `603`).

Reply:
Thanks for keeping a close eye on this critical piece of the wrapper code. I have updated the inline comments, hope that makes it clearer. I think the code is ok as-is.

Reply:
Ah, now it's clear! Thanks for explanation!